filament/src/ShadowMap.cpp

```diff
@@ -136,13 +136,15 @@ void ShadowMap::prepare(DriverApi& driver, SamplerGroup& sb) noexcept {
             TargetBufferFlags::DEPTH, dim, dim, 1,
             {}, { mShadowMapHandle }, {});
 
-    sb.setSampler(PerViewSib::SHADOW_MAP, {
-        mShadowMapHandle, {
-                    .filterMag = SamplerMagFilter::LINEAR,
-                    .filterMin = SamplerMinFilter::LINEAR,
-                    .compareFunc = SamplerCompareFunc::LE,
-                    .compareMode = SamplerCompareMode::COMPARE_TO_TEXTURE
-            }});
+    SamplerGroup::SamplerParams params;
+    params.filterMag = SamplerMagFilter::LINEAR;
+    params.filterMin = SamplerMinFilter::LINEAR;
+    params.compareMode = SamplerCompareMode::COMPARE_TO_TEXTURE;
+    params.compareFunc = SamplerCompareFunc::LE;
+
+	SamplerGroup::Sampler sampler { mShadowMapHandle, params };
```

Comment:
indentation

Reply:
fixed, there was a tab character on that line.